### ISSUE TITLE
Unremovable Duplicated Output Entities

### ISSUE THREAD

**Issue by warrenarnold** [gtdiehl/iotawatt_ha#36]
Hello, thanks for this integration, it makes getting the live values into Home Assistant a lot easier.

We have an issue with the 'outputs' coming through, the integration seems to create multiple entities for each output and it seems they never go away and can't be deleted even when the integration is removed.

![DuplicateSample](https://user-images.githubusercontent.com/74098948/146472177-a92f7fd0-ff99-4f64-abf7-d60a7cc41d64.PNG)

![image](https://user-images.githubusercontent.com/74098948/146472144-d311ab1a-2059-4e27-b43f-1f55b668b7de.png)

Can someone please look into why this would be happening ?

**Comment by OzGav** [comment]
The only thing I can think to ask is are you using MQTT?

**Comment by warrenarnold** [comment]
> 

Yes I do have MQTT running for some Tasmota Devices, would that interfere?

**Comment by OzGav** [comment]
Not unless it is those entities that are duplicating. MQTT can store entries for things that have been deleted and that causes them to reappear. However if you aren't using that for iotawatt then that isn't the problem.
If you keep re-adding the integration then that will be why you are getting multiple entities. You should be able to delete them from the CONFIGURATION menu?

**Comment by warrenarnold** [comment]
> Not unless it is those entities that are duplicating. MQTT can store entries for things that have been deleted and that causes them to reappear. However if you aren't using that for iotawatt then that isn't the problem. If you keep re-adding the integration then that will be why you are getting multiple entities. You should be able to delete them from the CONFIGURATION menu?

Thanks for your response, yes I have added it multiple times to try and fix the problem, and it keeps adding entities each time. I thought these would be deleted once the integration was removed ?

Which 'Configuration' menu are you referring to, I can't see that in HA?

I have 'System Options' which gives toggles for new entities and update polling, but no where to delete entities.

In entities it says this entity doesn't have a unique ID so it can't be edited/removed, as per pic above.

**Comment by OzGav** [comment]
This can be hard and might require editing the registry files. Do a Google search of “hassio remove entities” and you will get a bunch of threads and hopefully one will be the answer for you…
